web/src/main/java/org/mskcc/cbio/oncokb/api/pvt/PrivateSearchApiController.java

```diff
@@ -182,6 +190,108 @@ public ResponseEntity<LinkedHashSet<org.oncokb.oncokb_transcript.client.Drug>> s
         return new ResponseEntity<>(getLimit(new LinkedHashSet<>(oncokbTranscriptService.findDrugs(query)), limit), HttpStatus.OK);
     }
 
+     private List<TypeaheadSearchResp> findMatchingCancerTypes(String query, Boolean exactMatch) {
+        // for now only finding exact matches
+         // got all evidences and level
+         System.out.println("This got to cancer type search ");
+         Set<Evidence> evidences = EvidenceUtils.getEvidenceByEvidenceTypesAndLevels(EvidenceTypeUtils.getTreatmentEvidenceTypes(), LevelUtils.getPublicLevels());
+         Map<String, CancerTypeMatch> result = new HashMap<>();
+
+         query = query.toLowerCase();
+         System.out.println("This is the query: " + query);
+
+         ArrayList<TumorType> matchedTumorTypes = new ArrayList<>();
+
+         if (exactMatch){
+             matchedTumorTypes.add(ApplicationContextSingleton.getTumorTypeBo().getByName(query));
+         }
+
+         if (matchedTumorTypes.isEmpty() && !exactMatch) {
```

Comment:
It's essentially an else clause. 

```suggestion
         } else {
```